wicket-experimental/wicket-http2/wicket-http2-core/src/main/java/org/apache/wicket/http2/markup/head/PushHeaderItem.java

```diff
@@ -361,7 +366,32 @@ else if (url.toString().startsWith("."))
 					url = url.toString().substring(1);
 				}
 
-				urls.add(url.toString());
+				// The context path and the filter have to be applied to the URL, because otherwise
```

Comment:
Is someone able to review this and the following lines of code? I dislike the handling about the slashes in the url and to build up the path part that way.